docker/object_detector/Dockerfile

```diff
@@ -0,0 +1,71 @@
+##############################################################################
+# Base system packages and depedencies
+#
```

Comment:
I think we want to start with one general development node. We're still early to focus down into creating node-specific containers. We can use a general base note to run specific nodes in our built workspace.